src/20_Components/amp-bodymovin-animation.html

```diff
@@ -0,0 +1,55 @@
+<!-- 
+  ## Introduction
+
+  Embed Bodymovin animations in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-bodymovin-animation</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!--
+    Import the `amp-bodymovin-animation` component in the header.
+  -->
+  <script async custom-element="amp-brid-player" src="https://cdn.ampproject.org/v0/amp-bodymovin-animation-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-bodymovin-animation/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- ## Introduction -->
+  <!--
+    The `<amp-bodymovin-animation>` component embeds an <a href="http://airbnb.io/lottie/">AirBnB Bodymovin animation player</a>, which renders an animation from JSON generated by <a href="https://www.adobe.com/products/aftereffects.html">Adobe After Effects</a>.
+  -->
+
+  <!-- ## Basic Usage -->
+  <!-- **Example 1: Embed Bodymovin animations via the `src` attribute** -->
+  <amp-bodymovin-animation
+    layout="fixed"
+    width="200" height="200"
+    src="https://nainar.github.io/loader.json">
```

Comment:
Can you also copy the json source to the src/json directory? BTW, does the json file need to be served using AMP CORS headers?

Reply:
Done. Should I pull the JSON from the local files too? 

No it doesn't need to be served via AMP CORS headers. We do the fetching and the parsing of the JSON.

Reply:
Please pull the json from the local files. thanks!

Reply:
Done.